layers/+lang/typescript/packages.el

```diff
@@ -19,6 +20,17 @@
         web-mode
         ))
 
+(unless (or (configuration-layer/layer-usedp 'javascript)
+            (configuration-layer/layer-usedp 'html))
+  (defun typescript/init-add-node-modules-path ()
+    (use-package add-node-modules-path :defer t)))
```

Comment:
@TheBB Thanks. Will update later today. Where should this `web-common`/etc. layer go? Should it stay in `+lang`?